packages/context-editor/src/ContextEditor.tsx

```diff
@@ -96,14 +103,24 @@ function UnwrappedEditor(props: ContextEditorProps) {
 		const state = EditorState.create({
 			doc: props.initialDoc ? baseSchema.nodeFromJSON(props.initialDoc) : undefined,
 			schema: baseSchema,
-			plugins: basePlugins(
-				baseSchema,
-				props,
-				panelPosition,
-				setPanelPosition,
-				suggestData,
-				setSuggestData
-			),
+			plugins: [
+				...basePlugins(
+					baseSchema,
+					props,
+					panelPosition,
+					setPanelPosition,
+					suggestData,
+					setSuggestData
+				),
+				new Plugin({
```

Comment:
it's easier to include this plugin here rather than in `basePlugins` since this one needs access to `pluginViewFactory` which we get from the hook